app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -1,5 +1,51 @@
 <% @course_choices.each do |course_choice| %>
-  <div class='qa-application-choice-<%= course_choice.id %>'>
+  <% show_warning = course_choice.course_not_available? || course_choice.course_full? || course_choice.chosen_site_full? %>
+  <div class='qa-application-choice-<%= course_choice.id %> <%= show_warning ? 'review-warning'  : '' %>'>
+
+  <% if course_choice.course_not_available? %>
+    <div class='review-warning--primary-message'>
+      <p>You cannot apply to '<%= course_choice.course.name_and_code %>' because it is not running.</p>
+    </div>
+    <p class='govuk-body'><%= course_choice.provider.name %> have decided not to run this course.</p>
+    <p class='govuk-body'>You can:</p>
+    <ul class='govuk-list govuk-list--bullet'>
+      <li><%= link_to 'delete this choice', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %></li>
+      <li><%= link_to 'change to another course', course_change_path(course_choice), class: 'govuk-link', data: { action: :delete } %></li>
+      <li>
+        <%= link_to 'contact the training provider', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %>
```

Comment:
```suggestion
      <li><%= link_to 'change to another course', course_change_path(course_choice), class: 'govuk-link' %></li>
      <li>
        <%= link_to 'contact the training provider', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link' %>
```

These shouldn't be DELETE requests right?

Reply:
For the contact link, where should it go? It's currently pointing to `confirm_destroy`

Reply:
Copy/paste foolishness, updated with a proper contact link.